FIRSTFILE

```diff
@@ -0,0 +1,3 @@
+hi
+this is the first file i am creating
+it's really interesting
```

Comment:
@BiswajitMaharana ... I recommend added some bullets with great places to eat :smile: